integration/spark/README.md

```diff
@@ -72,61 +72,72 @@ spark = (SparkSession.builder.master('local').appName('rdd_to_dataframe')
-`spark.openlineage.url`
+The following parameters can be specified in the Spark configuration:
 
-NOTE: If `spark.openlineage.url` is defined, all the parameters will be taken from it regardless of whether they are
-also defined in configuration or not.
+### General
 
-#### Config entries
+Parameters configuring the Spark integration
 
-The following parameters can be specified in the Spark configuration:
+| Parameter                                | Definition                                                                                          | Example                         |
+------------------------------------------|-----------------------------------------------------------------------------------------------------|---------------------------------
+| spark.openlineage.transport.type         | The transport type used for event emit, default type is `console`                                   | http                            |
+| spark.openlineage.namespace              | The default namespace to be applied for any jobs submitted                                          | MyNamespace                     |
+| spark.openlineage.parentJobName          | The job name to be used for the parent job facet                                                    | ParentJobName                   |
+| spark.openlineage.parentRunId            | The RunId of the parent job that initiated this Spark job                                           | xxxx-xxxx-xxxx-xxxx             |
+| spark.openlineage.appName                | Custom value overwriting Spark app name in events                                                   | AppName                         |
+| spark.openlineage.facets.disabled        | `;` separated list of facets to disable, default is `spark_unknown;` (currently must contain `;`)   | spark_unknown;spark.logicalPlan |
 
-| Parameter | Definition | Example |
-------------|------------|---------
-| spark.openlineage.host | The hostname of the OpenLineage API server where events should be reported | http://localhost:5000 |
-| spark.openlineage.version | The API version of the OpenLineage API server | 1|
-| spark.openlineage.namespace | The default namespace to be applied for any jobs submitted | MyNamespace|
-| spark.openlineage.parentJobName | The job name to be used for the parent job facet | ParentJobName |
-| spark.openlineage.parentRunId | The RunId of the parent job that initiated this Spark job | xxxx-xxxx-xxxx-xxxx |
-| spark.openlineage.apiKey | An API key to be used when sending events to the OpenLineage server | abcdefghijk |
-| spark.openlineage.timeout | Timeout for sending OpenLineage info in milliseconds | 5000 |
-| spark.openlineage.appName | Custom value overwriting Spark app name in events | AppName |
-| spark.openlineage.url.param.xyz | A URL parameter (replace xyz) and value to be included in requests to the OpenLineage API server | abcdefghijk |
```

Comment:
this is used by many people, we can't simply remove it in one-go.

I would be really in favour of having a clear list of parameters that are deprecated. It should also contain a version when they're going to be removed.